src/main/java/org/jabref/model/openoffice/util/OOResult.java

```diff
@@ -0,0 +1,128 @@
+package org.jabref.model.openoffice.util;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+/*
+ * error cannot be null
+ * result cannot be null
```

Comment:
Internal variable, please comment at the variables

Reply:
I thought of these more like stating class requirements.
`error` and `result` here can also be understood as constructor parameters.

Replaced with

```
An instance of this class represents either the result
of a computation, or an error value. Neither of these is allowed
to be null.
```